thirty-five-mm/app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,97 @@
  *= require_tree .
  *= require_self
  */
+
```

Comment:
This css should be in a separate file - application.css is intended to be a manifest file.